crml/transaction-payment/src/lib.rs

```diff
@@ -438,14 +438,79 @@ where
 		}
 	}
 
+	/// Introspectable version of `compute_fee_raw`.
+	/// Used to inspect fee composition in runtime integration tests, `tip` is ignored.
+	pub fn compute_fee_parts(len: u32, info: &DispatchInfoOf<T::Call>) -> FeeParts<BalanceOf<T>>
```

Comment:
Would #[cfg(test)] be needed here to silent a compiler warning?

Reply:
Couldn't get it to work for some reason, will have another go in the next PR